content/en/brupop/1.3.x/setup/configure/index.markdown

```diff
@@ -0,0 +1,280 @@
++++
+title = "Configure Brupop"
+type = "docs"
+description = "Making the operator work for your needs" 
+weight = 30
++++
+
+When you install Brupop, the operator comes pre-configured with reasonable defaults.
+[Labeling your nodes](#label-nodes) is the only required configuration step.
+
+Aside from labeling nodes, you configure Brupop with [helm](https://helm.sh/) or with a manifest.
```

Comment:
```suggestion
Aside from labeling nodes, you can configure Brupop with [helm](https://helm.sh/) or with a manifest.
```

Reply:
Using 'can' turns this into a conditional, the intention is to make this unconditional.